pkg/controller/registration/registration_controller.go

```diff
@@ -196,44 +192,17 @@ func registerDefaultController(r *ReconcileRegistration, config *controller.Conf
 		JitterGen:    r.jitterGenerator,
 		Defaulters:   r.defaulters,
 	}
-
 	var schemaUpdater k8s.SchemaReferenceUpdater
 	if kccfeatureflags.UseDirectReconciler(gvk.GroupKind()) {
```

Comment:
Just a note for reviewers: The previous code only turn on LLM in direct controller, all other resources that already have direct controller and API are only turning on via the `KCC_USE_DIRECT_RECONCILERS` flag (and those part is not covered by the e2e tests). 
When those resources are already, we can simply remove the `"cnrm.cloud.google.com/tf2crd=true"`  label from their CRD.

cc @justinsb @acpana

Reply:
Cool!